bindings/nodejs/src/lib.rs

```diff
@@ -33,20 +31,40 @@ pub struct OperatorFactory {}
 impl OperatorFactory {
     #[napi]
     pub fn memory() -> Result<Operator> {
-        let op = opendal::Operator::create(services::Memory::default())
+        let op = opendal::Operator::create(opendal::services::Memory::default())
             .unwrap()
             .finish();
 
         Ok(Operator::new(op))
     }
 }
 
-#[allow(dead_code)]
 #[napi]
-pub struct ObjectMeta {
-    pub location: String,
-    pub last_modified: i64,
-    pub size: u32
+pub struct Memory {}
+
+#[napi]
+impl Memory {
+    #[napi(constructor)]
+    pub fn new() -> Self {
+        Self {}
+    }
+
+    #[napi]
+    pub fn build(&self) -> Operator {
+        self.make_operator()
+    }
+}
```

Comment:
How about API like this?
Users can import the service they need, set config, and finally use the build function to get an Operator.

Reply:
WDYT @Xuanwo

Reply:
It is common for Node.js projects to initialize their structures using a format such as `new({a: xx, b: yy})`. As we work on developing bindings for OpenDAL in multiple languages, we need to maintain a balance between the Rust API and the stylistic preferences of these languages. 

Anyway, this is not a final decision and we still have space to imporve them. Please pick your favourite style and keep going.

Reply:
My expection: users feel like it's a native lib witten in nodejs :rocket:

Reply:
> My expection: users feel like it's a native lib written in nodejs 🚀

I think the current design looks quite similar to a native JavaScript library~

Reply:
> I think the current design looks quite similar to a native JavaScript library~

Let's go! You are the code owner!